examples/test_case_globcurrent.py

```diff
@@ -0,0 +1,40 @@
+from parcels import Grid, Particle, JITParticle, AdvectionRK4
+from datetime import timedelta as delta
+import pytest
+
+
+def set_globcurrent_grid():
+    filenames = {'U': "examples/GlobCurrent_example_data/20*-GLOBCURRENT-L4-CUReul_hs-ALT_SUM-v02.0-fv01.0.nc",
+                 'V': "examples/GlobCurrent_example_data/20*-GLOBCURRENT-L4-CUReul_hs-ALT_SUM-v02.0-fv01.0.nc"}
+    variables = {'U': 'eastward_eulerian_current_velocity', 'V': 'northward_eulerian_current_velocity'}
+    dimensions = {'lat': 'lat', 'lon': 'lon',
+                  'time': 'time'}
+    return Grid.from_netcdf(filenames, variables, dimensions)
+
+
+def test_globcurrent_grid():
+    grid = set_globcurrent_grid()
+    assert(grid.U.lon.size == 81)
+    assert(grid.U.lat.size == 41)
+    assert(grid.U.data.shape == (365, 41, 81))
+    assert(grid.V.lon.size == 81)
+    assert(grid.V.lat.size == 41)
+    assert(grid.V.data.shape == (365, 41, 81))
+
+
+@pytest.mark.xfail(run=False, reason="32/64 bit bug in time (github PR #83)")
```

Comment:
Just did that, and also changed file names to example_* and tox.ini to setup.cfg. See 188c611